tests/net/quay_io/mod.rs

```diff
@@ -240,6 +240,32 @@ fn test_quayio_has_no_manifest() {
     assert_eq!(has_manifest, None);
 }
 
+#[test]
+fn test_quayio_auth_manifestref_missing() {
+    let image = "steveej/cincinnati-test";
+    let tag = "no-such-tag";
+
+    let login_scope = format!("repository:{}:pull", image);
+    let (mut runtime, dclient) = common_init(Some(&login_scope)).unwrap();
+    let fut_actual = async { dclient.get_manifestref(image, tag).await.unwrap() };
```

Comment:
This actually returns an `Err` as the tag does not exist. I think you wanted to `.is_err()` somewhere around it.